.gitignore

```diff
@@ -32,3 +31,5 @@ fileeditor
 *.iws
 .idea/
 out/
+
+**/generated_src/
```

Comment:
what test would I add for this? not sure if it's worth future proofing git ignore file